express-mutations/server.ts

```diff
@@ -83,8 +83,9 @@ app.delete('/api/actors/:actorId', async (req, res, next) => {
     const result = await db.query(sql, params);
     const actor = result.rows[0];
     if (!actor) {
-      throw new ClientError(404, ` actor ${actorId} does not exist`);
+      throw new ClientError(204, ` actor ${actorId} does not exist`);
```

Comment:
Ooops, this one should be a 404. You send a 404 if the actor doesn't exist. You send a 204 if you successfully deleted the actor.

Reply:
i changed that on accident, forgot to change it back, whoops